rules/creates-intercom-user.md

```diff
@@ -0,0 +1,53 @@
+---
+gallery: true
+short_description: On first login/signup call the Intercom API to record the contact as a new User
+categories:
+- webhook
+---
+## Creates a new user in Intercom on First Login/SignUp
+
+This rule will check if this is the first login of the user, and will call the Intercom API to save the contact as a new user. A persistent property named `signedUp` is used to track whether this is the first login or not.
+
+```
+function (user, context, callback) {
+  var moment = require('moment-timezone');
+  user.app_metadata = user.app_metadata || {};
+  
+  // Only create a new intercom user when user signs up
+  if (!user.app_metadata.signedUp) {
```

Comment:
Could you look at this issue?
https://github.com/auth0/rules/issues/102

I think we ought to use a property name more specific to this Rule, such as `userSentToIntercom` or `_intercom_signup` if we want to rely on metadata (i.e. for fault-tolerance as I've commented below).